mount.go

```diff
@@ -16,5 +16,78 @@
 
 package virtcontainers
 
+import (
+	"fmt"
+	"path/filepath"
+	"syscall"
+)
+
 // These mounts need to be created by the agent within the VM
 var systemMounts = []string{"/proc", "/dev", "/dev/pts", "/dev/shm", "/dev/mqueue", "/sys", "/sys/fs/cgroup"}
+
+func major(dev uint64) int {
+	return int((dev >> 8) & 0xfff)
+
+}
+
+func minor(dev uint64) int {
+	return int((dev & 0xff) | ((dev >> 12) & 0xfff00))
+}
+
```

Comment:
@sameo The above code does not seem to give the correct major-minor numbers. I know thats the how its documented in libc. This is what I followed:
https://github.com/golang/go/issues/8106